### [Chilrden's Hospital of Philadelphia](https://www.chop.edu/) (CHOP)

The CHOP Comprehensive Solid Tumor Panel and Comprehensive Hematologic Cancer Pane include sequence and copy number analyses of 238 and 117 cancer genes, respectively, genotyping of two genes associated with cancer pharmacogenomics, and a Fusion Panel targeting over 700 exons of 117 cancer genes.

Next generation sequencing (NGS) and data analysis: Nucleic acid is extracted from the patientâ€™s sample following standard DNA and RNA extraction protocols. Extracted DNA is fragmented and tagged using SureSelect QXT target enrichment to generate adapter-tagged libraries. Biotin-labeled probes specific to the targeted regions are used for capture hybridization. Libraries are enriched for the desired regions using streptavidin beads. Enriched libraries are then indexed and pooled for sequencing. Libraries are subject to sequence analysis on Illumina NovaSeq 6000 system for 150 bp paired end reads. All coding exons and the flanking intron sequences of targeted genes in the panel are sequenced, and selected promoter regions and known intronic variants are also evaluated. Sequence data are analyzed using the home brew software ConcordS V4.0.0 and NextGENe V2 NGS Analysis Software. Sequence variants within exons and 5 bp flanking intron sequences are annotated. Copy number variation (CNV) analysis for gross deletions and duplications are evaluated using NGS data. RNA sequencing libraries are prepared using Archer Universal RNA Reagent Kit with CHOP fusion panel custom-designed primers with target specific molecular barcode. Sequencing data are analyzed using ArcherTM Analysis for fusion genes. Clinically significant variants including single nucleotide variants (SNVs), indels, CNVs and fusion genes are confirmed by Sanger sequencing, MLPA, Real-Time PCR, or ddPCR only when necessary.

Variant categorization and reporting: Sequence variants, copy number variants, and gene fusions are evaluated based on the currently available information from relevant resources, such as professional guidelines, clinical and population variant databases, tumor specific databases, and the scientific literature. Sequence variants are reported according to HGVS nomenclature [den Dunnen 2016, PMID: 26931183]. Somatic variants are classified using criteria consistent with those recommended by the Association for Molecular Pathology (AMP), American Society of Clinical Oncology (ASCO), and College of American Pathologists (CAP) [Li 2017, PMID: 27993330], as described below. Tier 1-3 variants are reported. Tier 4 variants are not reported.

TPMT and NUDT15 genotyping: NUDT15 diplotypes *1 through *9, and TPMT diplotypes *1, *2, *3A, *3B, and *3C, are assessed [Relling 2019, PMID: 30447069; CPIC Guideline for Thiopurines and TPMT and NUDT15].
