### [The University of Chicago](https://www.uchicago.edu/) (UCHI)

The University of Chicago submitted data in the current data set includes somatic variants (SNVs and INDELS) identified using two amplicon-based targeted tumor-only assays. The ONCOSCREEN assay is tumor only targeted gene panel covering 50 genes. The ONCOHEME assay is a tumor only targeted gene panel covering 53 genes. Both ONCOSCREEN and ONCOHEME assays follow the same DNA extraction, data processing, alignment and variant calling steps. Only ONCOSCREEN assay has sample replicates (The samples were prepped and sequenced in duplicate and the variant calls were reported only if the variant was detected in both replicates). DNA is extracted from unstained sections of FFPE tissue paired with an H&E stained section that is used to ensure adequate tumor cellularity (human assessment > 20%) and marking of the tumor region of interest (macrodissection). No paired "normal" DNA is extracted from each case. The Novoalign v3.02.07 aligner (NovoCraft, Selangor, Malaysia) was used to align to hg19 using automatic adapter and primer trimming options for paired end 2x152bp reads sequencing for ONCOSCREEN assay and paired end 2x255bp reads sequencing for ONCOHEME assay on the Illumina Miseq platform. The input is a .fastq file and the output is a .sam file. Before alignment, off-target reads are removed from the FASTQ files by matching the primer pairs of the amplicons to the reads. After alignment, we use picard tools v1.92 to convert the .sam file to .bam file. samtools v0.1.19 mpileup is run on the aligned .bam file. Custom python scripts process the output of samtools mpileup for each sample to identify SNV & INS/DEL. Details in http://www.sciencedirect.com/science/article/pii/S1525157816301945. The filtered FASTQ files were analyzed for the presence of insertion and deletion mutations >5bp using the reference-independent Amplicon Indel Hunter (https://www.sciencedirect.com/science/article/pii/S1525157815001506). Variant filtering criteria was applied. For ONCOSCREEN assay, the cutoff is as follows: Read Depth with Phred score (>Q30) >= 200 and Variant allele frequency - SNV >= 0.05, INS/DEL >= 0.05. The reported depth is that of bases with Phred quality score >30.  Depth and allele frequency values of the replicates were averaged for reporting. For ONCOHEME assay the cutoff is as follows: Read Depth with Phred score (>Q30) >= 100 and Variant allele frequency - SNV >= 0.05, INS/DEL >= 0.05. The reported depth is that of bases with Phred quality score >30. No filters for data uploaded to GENIE for both assays.  
